.github/workflows/test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test Setup
+
+on:
+  pull_request:
+    branches:
+      - develop
+  push:
+    branches:
+      - develop
+      - trunk
+
+jobs:
+  test:
+    name: ${{ matrix.core.name }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        core:
+          - { name: "WP latest", version: "latest" }
+          - { name: "WP 4.9.19", version: "4.9.19" }
```

Comment:
For some reason the 4.9.19 test is consistently failing, see [the action's log](https://github.com/10up/cypress-wp-setup/runs/5457155364).

The tag appears to be correct for WordPress/WordPress so maybe you need to do a fetch or something?

Reply:
Thanks @peterwilsoncc, I also noticed this. It's strange because this particular test [has passed during previous run](https://github.com/10up/cypress-wp-setup/runs/5448504121?check_suite_focus=true).

Checking artifacts to find why does if fail this time...